pfsimulator/parflow_lib/metadata.c

```diff
@@ -0,0 +1,699 @@
+#include "parflow.h"
```

Comment:
Please include GNU license and optional copyright statement if one is needed by your organization.